### PR TITLE
Added RTL support to SideMenu

### PR DESCRIPTION
**Pull request by rghorbani**
### What changed

Added rtl support to SideMenu.NavigationBadge

### Why it changed

...

---

- [x] Change is tested
- [x] Change is documented
- [x] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by shlomitc** [comment]
Thanks for contributing!

Can you please add an example to the story page with the `rtl` example? The story file is `stories/SideMenu/index.js`

Just render the example one more time with a `<div className="rtl"/>`.
Having the regular and the RTL one next to each other will be great.

Also - can you please remove the changelog addition? I will do that after merging.

Thanks!

**Comment by rghorbani** [comment]
@shlomitc Did the changes you requested.

- Added Sidemenu RTL examle
- Reverted CHANGELOG

Thank You

File: stories/SideMenu/index.js [hunk 1]
```diff
@@ -34,10 +34,16 @@ storiesOf('6. Navigation', module)
 
       <h1>Usage examples</h1>
 
-      <CodeExample title="Standard" code={ExampleStandardRaw}>
+      <CodeExample title="LTR Standard" code={ExampleStandardRaw}>
```
[the remark this answers is not in the document]
Comment (rghorbani) [reply stories/SideMenu/index.js]: removed.

**Comment by rghorbani** [comment]
@shlomitc Fixed rest of the component as well.
I also changes the title, as it seems more appropriate.

File: stories/SideMenu/index.js [hunk 2]
```diff
@@ -38,6 +38,12 @@ storiesOf('6. Navigation', module)
         <ExampleStandard/>
       </CodeExample>
 
+      <div className="rtl" style={{direction: 'rtl'}}>
```
Comment (shlomitc) [inline stories/SideMenu/index.js]: Better put this `div` inside the code example and not outside of it, otherwise these such things happen
image

**Comment by rghorbani** [comment]
@shlomitc Added new file for RTL example to fix the code view, As you requested.

**Pull request merged**